q2_fmt/_peds.py

```diff
@@ -216,49 +184,49 @@ def sample_peds(table: pd.DataFrame, metadata: qiime2.Metadata,
 
     with warnings.catch_warnings():
         warnings.simplefilter("ignore")
-        peds = median_numerator_series/median_denominator_series
+        pedf = median_numerator_series/median_denominator_series
 
-    peds_df['measure'] = peds
-    peds_df['transfered_donor_features'] = median_numerator_series
-    peds_df['total_donor_features'] = median_denominator_series
-    peds_df = peds_df.reset_index()
+    pedf_df['measure'] = pedf
+    pedf_df['transfered_donor_features'] = median_numerator_series
+    pedf_df['total_donor_features'] = median_denominator_series
+    pedf_df = pedf_df.reset_index()
 
-    peds_df['id'].attrs.update({
+    pedf_df['id'].attrs.update({
         'title': metadata_df.index.name,
         'description': 'Sample IDs'
     })
-    peds_df['measure'].attrs.update({
-        'title': "Sample PEDS",
+    pedf_df['measure'].attrs.update({
+        'title': "Sample pedf",
```

Comment:
```suggestion
        'title': "Sample PEDF",
```
Should this be in all caps, like PEDS was before?